en/flying/missions.md

```diff
@@ -18,6 +18,28 @@ For more information see the [QGroundControl User Guide](https://docs.qgroundcon
 
 ![planning-mission](../../assets/flying/planning_mission.jpg)
 
+### Mission Feasibility Checks
+
+Upon receiving an updated mission file from the ground station, the flight controller runs some feasibility checks that are meant to ensure the safety of the vehicle and catch unintended mission settings. If one of the checks fails, the user is notified and it is not possible to start the mission.

```

Comment:
My understanding is that checks are run on more than just upload. I.e. I could upload a mission and the vehicle fails the checks, I could then drive to the place where the mission is supposed to be used and it passes the checks.

Point being that the checks must be run again at some point - I am assuming they are run every time you switch to mission mode? We should make it clear.

Reply:
True, the checks are also run once after boot as soon as the home position is valid. Switching to Mission doesn't seem to be necessary.

I'm though not even sure if a rejected mission is stored on the vehicle, so if your example of the checks failing at location A, then passing automatically at location B would work. 

@Igor-Misic  weren't you involved in the building the logic around there? Would you know if this example is possible like that?

Reply:
@sfuhrer @Igor-Misic THanks very much. I have fixed it up to say this (I think) in the commit https://github.com/PX4/PX4-user_guide/pull/2226/commits/a3376e34612faff32f1a024c8265de55ebe8108a

There is a bit of a restructure, but the main effect is that it now says this:

> The checks are run when the mission is uploaded and immediately before it is run.
> If any of the checks fail, the user is notified and it is not possible to start the mission.

It also says that if the mode is switched to but not allowed to start due to being infeasible, then this is the same as if there is no mission when you switch to the mode - the vehicle holds/loiters.

Reply:
Thanks @sfuhrer - I will revert that then. Probably in most cases that will be sufficient - i.e. you tend to boot when you're ready to fly, so that is when making the check makes most sense.

Reply:
Actually, I see you already reverted.